odoo/addons/base/views/res_partner_views.xml

```diff
@@ -186,14 +186,9 @@
 
                     <group>
                         <group>
-                            <field name="type" groups="base.group_no_one" attrs="{'invisible': [('is_company','=', True)], 'readonly': [('user_ids', '!=', [])]}"/>
                             <span class="o_form_label o_td_label" name="address_name">
-                                <b attrs="{'invisible': ['|', ('parent_id', '=', False), ('type', '!=', 'contact')]}">Company Address</b>
-                                <b attrs="{'invisible': ['|', ('parent_id', '=', False), ('type', '!=', 'invoice')]}">Invoice Address</b>
-                                <b attrs="{'invisible': ['|', ('parent_id', '=', False), ('type', '!=', 'delivery')]}">Delivery Address</b>
-                                <b attrs="{'invisible': ['|', ('parent_id', '=', False), ('type', '!=', 'other')]}">Other Address</b>
-                                <b attrs="{'invisible': ['|', ('parent_id', '=', False), ('type', '!=', 'private')]}">Private Address</b>
-                                <b attrs="{'invisible': ['&amp;', ('parent_id', '!=', False), ('type', '!=', False)]}">Address</b>
+                                <field name="type" groups="base.group_no_one" attrs="{'invisible': [('is_company','=', True)], 'required': [('is_company','!=', True)], 'readonly': [('user_ids', '!=', [])]}" class="font-weight-bold"/>
+                                <b class="position-absolute" attrs="{'invisible': [('is_company', '=', False)]}">Address</b>
```

Comment:
Is there any reason why this tag has the `position-absolute` class?

Actually, it makes the address unreadable (cf. text overlapping) on smaller screens/mobile... and I don't see the point on desktop 😕

Reply:
![image](https://user-images.githubusercontent.com/43335401/135420749-b1853209-8717-4591-a04a-ece504cc3d3c.png)

Reply:
I will see if I can fix it in https://github.com/odoo/odoo/pull/77344

Reply:
Everything seems to work fine without position absolute, I don't remember why I added it...

Fixed in the PR mentioned above.

Reply:
Thanks @william-andre :smile: